VAMobile/documentation/docs/Engineering/BackEnd/Architecture/Auth Diagrams.md

```diff
@@ -14,9 +15,10 @@ sequenceDiagram
     App-->>User: Display auth result
 ```
 
+### Biometrics Flow for Username and Password Initial Login
```

Comment:
I added markdown titles to each graph so that it's more obvious when there is a rendering error. I spent over an hour trying to prevent the rendering error but had no luck. From what I can tell, there is a conflict with the IDs of the mermaid graphs and there isn't a way to hardcode unique IDs to the graphs. At least with these titles on the page, you can refresh until each section is populated

Reply:
I think you might be able to solve this using aliases. https://mermaid.js.org/syntax/sequenceDiagram.html#aliases

Reply:
If not I would just screenshot the diagrams and add them as images to avoid the reloading issue.

Reply:
> It looks like aliases are for actors though, not the chart itself. Unless I'm misunderstanding the docs this wouldn't help, unless you're thinking similarly named actors are what's causing the charts to freak out?

I thought the actors could be causing the issue. Worth trying or just using images

Reply:
Sadly it did not fix the issue. I updated with screenshots and left the code blocks so that we can regenerate them as needed